### PR TITLE
added folder management and popular movies page

### PR DESCRIPTION
**Pull request by marcin10lw**
I had to change MovieTile styling to be more flexible, cause when we had more that 3 Tags it's layout was breaking 🙂
Also I've organized our file management to be more to make it more react router-friendly

### CODE REVIEW

**Review by HelkaPL — APPROVED** [review]

**Comment by marcin10lw** [comment]
Our Header looked like this, so i gave TextTitle and StyledVideoIcon `flex-shrink: 0`
image

Also a small gap occured between search icon and input so I gave `margin-right: -0.7px` to SearchBarIcon

File: src/features/movies/PopularMoviesPage/index.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+const PopularMoviesPage = () => {
+  const base_url = "https://api.themoviedb.org/3";
+  const api_key = "api_key=0e017d1cf3aa44c76887c24c592892f0";
+
+  const [popularMovies, setPopularMovies] = useState([]);
+
+  useEffect(() => {
+    fetch(`${base_url}/discover/movie?sort_by=popularity.desc&${api_key}`)
+      .then((response) => response.json())
+      .then((data) => setPopularMovies(data.results));
```
Comment (mikolajgraczyk) [inline src/features/movies/PopularMoviesPage/index.js:15]: I think we can remove unnecessary bracket

obraz

File: src/features/movies/PopularMoviesPage/index.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  }, []);
+
+  return (
+    <Container>
+      <TilesSection title="Popular movies">
+        {popularMovies &&
+          popularMovies.map((movie) => (
```
Comment (mikolajgraczyk) [inline src/features/movies/PopularMoviesPage/index.js:22]: also here 
obraz

**Review by mikolajgraczyk — APPROVED** [review]

File: src/features/movies/PopularMoviesPage/index.js (hunk 2 above)
Comment (marcin10lw) [reply src/features/movies/PopularMoviesPage/index.js:22]: Same formatting thing

File: src/features/movies/PopularMoviesPage/index.js (hunk 1 above)
Comment (marcin10lw) [reply src/features/movies/PopularMoviesPage/index.js:15]: It's a feature with prettier formatter

**Pull request merged**